.travis.yml

```diff
@@ -2,9 +2,6 @@ language: python
 python:
 - 2.7
 - 3.5
-
-sudo: "required"
```

Comment:
I think the default also depends on the age of the project, so projects that pre-date container based infrastructure actually default to true.